### ISSUE TITLE
Todo with hierarchy support 

### ISSUE THREAD

**Issue by bhujoshi** [upnotes-io/react-todo#5]
- User should be able to create todo hierarchies.

**Comment by Okewusi** [comment]
@bhujoshi is this still open?

**Comment by bhujoshi** [comment]
Yes @Okewusi. Assigning to you let me know in case any help needed form me.

**Comment by davemarong** [comment]
This seems interesting. Is this still open?

**Comment by bhujoshi** [comment]
@davemarong Yes it is open you can pick this up. Let me know in case any help needed thanks

**Comment by Trevy05** [comment]
is this still open?

**Comment by kamalkishor1991** [comment]
@Trevy05 yes it is still open.

**Comment by Trevy05** [comment]
could you explain the deliverables more clearly, I would like to be assigned this issue.

**Comment by kamalkishor1991** [comment]
@Trevy05 thanks for picking this up.
Something like this with support for multiple levels:
![image](https://user-images.githubusercontent.com/5221843/179334925-e724daa6-d395-4e96-821f-27c7d392d97c.png)

1. User should be able to move items with tab to make something child of previous item based on the level previous child is at.
2. drag and drop should work 
you can take reference of https://keep.google.com/ and use checkboxes feature.
Please let me know if you have more questions? Assigning it to you.

**Comment by Kirito64** [comment]
is it still open?

**Comment by ABay23** [comment]
Can I work on this issue?

**Comment by kamalkishor1991** [comment]
@ABay23 sure.

**Comment by bhujoshi** [comment]
Yeah,  I think `+ add item ` should only come on hover. Also, the placeholder name can be replaced with `+ add child item` with more padding on the left. Let me know in case you need any help.

**Comment by Curis-lab** [comment]
that's will be lovely, I like your idea, thanks..
